src/main/views/summary-of-publications.njk

```diff
@@ -0,0 +1,52 @@
+{% from "./macros/common-components.njk" import goBack %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% extends "template.njk" %}
+{% block pageTitle %}
+  {{ title }}
+{% endblock %}
+{% block beforeContent %}
+  {{ super() }}
+  {{ goBack() }}
+{% endblock %}
+{% block content %}
+  <h1 class="govuk-heading-l">{{ title }} {{ courtName }}</h1>
+  {% if publications.length === 0 %}
+    <p class = "govuk-body govuk-!-font-size-16">Sorry, no lists found for this court</p>
+  {% else %}
+  <h2 class="govuk-body">{{ p1 }}</h2>
+
+
+  <div class="parent-box">
+  {% set pubs = [] %}
+  <p class = "govuk-body govuk-!-font-size-16">
+
+  {% for item in publications |sort(reverse= true, attribute="contentDate, listType")%}
+
+  <a class = das-search-results__link href = {{item.artefactId}}>{{ item.listType | listType }}
+  {{item.contentDate | date("DD MMMM YYYY")}}</a>
+  {% if item.isFlatFile %}
+  (opens in a new window)
+  {% endif %}
+
+  <br>
```

Comment:
can we extend with the margin classes at all or does it have to be breaks?

Reply:
<img width="447" alt="image" src="https://user-images.githubusercontent.com/18507008/152648493-f7a96f3c-db6c-43aa-a34b-9f198eea4f96.png">
I was just building off the heroku implementation. Happy to do it with CSS instead if that's desired!

Reply:
if possible yeh that would be neater